app/index.html

```diff
@@ -2,15 +2,32 @@
 <html>
 <head>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-  <link type="text/css" href="font-awesome/css/font-awesome.min.css" rel="stylesheet"> 
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link type="text/css" href="font-awesome/css/font-awesome.min.css" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Cabin:400,600" rel="stylesheet"> 
   <link type="text/css" href="styles/main.css" rel="stylesheet" media="screen">
   <title>Headers</title>
 </head>
 <body>
   <h1>HELLO WORLD!</h1><i class="fa fa-hand-spock-o" aria-hidden="true"></i>
   <div class="wrapper">
   	<div class="header header__first"></div>
-  	<div class="header header__second"></div>
+  	<div class="header header-second">
+      <div class="header-second-content">
+         <label for="header-second__bars" class="header-second__bars fa fa-bars" aria-hidden="true">
+          </label>
+           <input class="header-second__check" id="header-second__bars" type="checkbox">
+         <nav class="nav header-second__nav">
+            <a href="#" class="nav__link">Products</a>
```

Comment:
please make a `ul` `li` list  with `a` tags inside.

Reply:
![image](https://user-images.githubusercontent.com/33283739/34075582-b9461e88-e2d3-11e7-8995-7cce520d914e.png)

why not do it like I did? Here, for example, as well as me

Reply:
I saw Yura (our teacher) do the same as this Yura. Maybe we need to discuss this on the next lesson too.

Reply:
Using `ul` and `li` are more semantic for menu as I know